### PR TITLE
[Wallet] Fix disconnecting banner overlap in wallet home

### LINKED ISSUE #1286: [Android] Error “Bad Connection...” is overlaps on QR code icon when device internet connection is bad.
**App version:** Play store build v1.5.0
**Repro on:** Moto E (4.4.4),Samsung  Galaxy S4 (5.0),  Samsung Galaxy A5 (6.0.1). 
**Does not repro:** Samsung j7 prime (6.0.1), Samsung Galaxy A8+ (8.0.0), Samsung galaxy note 9 (9.0)
**Priority:** Low
**Testing Account:** (315) 678-6563/[Backup Kay](https://docs.google.com/spreadsheets/d/1Iy41GGA6e2pES2XHjBn7nb8X3n8zZsu7EW1jvwD2e70/edit?usp=sharing)
 
**Repro Steps:** 
1) Launch the app. 
2) Turn off the device internet connection and the user is on the Wallet tab. 
3) Make sure Bad connection error message is shown. 
4) Observe. 

**Current Behavior:** Error “Bad Connection...” overlaps on QR code icon
**Expected Behavior:** Error “Bad Connection...” should not overlaps on QR code icon
**Attachment:** Android_SmallScreen_BadConnectionErrorOvserlapIssue.png
Android_SmallScreen_BadConnectionErrorOvserlapIssue

### PR DESCRIPTION
**Pull request by annakaz**
### Description

Tiny PR fixing the disconnected banner overlap in wallet home. This does not affect the UI when the banner is not shown 

### Tested

Added and ran jest tests

### Related issues

- Fixes #1286 

Previously: 
66563704-00004300-eb7c-11e9-9584-ec53a25de9dd

Now: 
Screenshot_20191115-160300

### CODE REVIEW

**Comment by jeanregisser** [comment]
The "Connecting to Celo..." is also moved down a little, right?

**Review by jeanregisser — APPROVED** [review]
LGTM! Thanks for fixing this! 🎉 
I wish we had a less invasive UI connection status element. 👀

File: packages/mobile/src/home/WalletHome.test.tsx [hunk 1]
```diff
@@ -54,4 +54,64 @@ describe('Testnet banner', () => {
...
+          refreshAllBalances={jest.fn()}
+          resetStandbyTransactions={jest.fn()}
+          initializeSentryUserContext={jest.fn()}
+          exitBackupFlow={jest.fn()}
+          setLoading={jest.fn()}
+          showMessage={jest.fn()}
+          hideAlert={jest.fn()}
+          importContacts={jest.fn()}
+          loading={false}
+          appConnected={false}
```
Comment (jmrossy) [inline packages/mobile/src/home/WalletHome.test.tsx:71]: I think this value gets overwritten by the value set in the provider store. I don't see the banner in the snapshot.

Comment (annakaz) [reply packages/mobile/src/home/WalletHome.test.tsx:71]: Good catch. Updated to use `createMockStoreAppDisconnected()`.

**Pull request merged**